src/index.html

```diff
@@ -17,16 +17,20 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form action="post">
+      <input
+        class="input big-input"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class name 'big-input' seems to describe the style rather than the content or function of the input.